### DISCUSSION TITLE
Missing sensor

### DISCUSSION THREAD

**Discussion by blaired** [BottlecapDave/HomeAssistant-OctopusEnergy#270] [Q&A] [answered]
I’ve recently had a smart meter installed, I can see my consumption in the app OK (Octopus) but can’t find the below sensor when trying to setup consumption in the energy dashboard 

consumed energy
 sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_consumption

**Comment by BottlecapDave** [comment, marked as answer]
Hi there. Do you have a home mini and checked the box in your account settings for the integration? As per the instructions https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/energy_dashboard.md#current-consumption this sensor is only available with this device.

**Reply by blaired** [reply]
Hi, no I don’t I’m on the waiting list. I’m guessing I need this for real time monitoring?

**Reply by BottlecapDave** [reply]
Correct. As per the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_account.md#home-mini) those sensors are only available with that additional device. This is due to how smart meters work and how often they send the data to Octopus Energy
